.pre-commit-config.yaml

```diff
@@ -18,10 +18,6 @@ repos:
           - --remove-unused-variables
           - --remove-all-unused-imports
           - --expand-star-imports
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
-    hooks:
-      - id: flake8
```

Comment:
what is the significance of this removal?

Reply:
just removed unnecessary pre-commit checks, but tbh this is unrelated to this PR.